types/deno/index.d.ts

```diff
@@ -0,0 +1,9756 @@
+        /** Estimation of how much memory, in bytes, is available for starting new
+         * applications, without swapping. Unlike the data provided by the cache or
+         * free fields, this field takes into account page cache and also that not
+         * all reclaimable memory will be reclaimed due to items being in use.
+         */
+        available: number;
+        /** Memory used by kernel buffers. */
+        buffers: number;
+        /** Memory used by the page cache and slabs. */
+        cached: number;
+        /** Total swap memory. */
+        swapTotal: number;
+        /** Unused swap memory. */
+        swapFree: number;
+    }
+
+    /** Reflects the `NO_COLOR` environment variable at program start.
+     *
+     * When the value is `true`, the Deno CLI will attempt to not send color codes
+     * to `stderr` or `stdout` and other command line programs should also attempt
+     * to respect this value.
+     *
+     * See: https://no-color.org/
+     *
+     * @category Runtime
+     */
+    export const noColor: boolean;
+
+    /**
+     * Returns the release version of the Operating System.
+     *
+     * ```ts
+     * console.log(Deno.osRelease());
+     * ```
+     *
+     * Requires `allow-sys` permission.
+     * Under consideration to possibly move to Deno.build or Deno.versions and if
+     * it should depend sys-info, which may not be desirable.
+     *
+     * @tags allow-sys
+     * @category Runtime
+     */
+    export function osRelease(): string;
+
+    /**
+     * Returns the Operating System uptime in number of seconds.
+     *
+     * ```ts
+     * console.log(Deno.osUptime());
+     * ```
+     *
+     * Requires `allow-sys` permission.
+     *
+     * @tags allow-sys
+     * @category Runtime
+     */
+    export function osUptime(): number;
+
+    /**
+     * Options which define the permissions within a test or worker context.
+     *
+     * `"inherit"` ensures that all permissions of the parent process will be
+     * applied to the test context. `"none"` ensures the test context has no
+     * permissions. A `PermissionOptionsObject` provides a more specific
+     * set of permissions to the test context.
+     *
+     * @category Permissions */
+    export type PermissionOptions =
+        | "inherit"
+        | "none"
+        | PermissionOptionsObject;
+
+    /**
+     * A set of options which can define the permissions within a test or worker
+     * context at a highly specific level.
+     *
+     * @category Permissions */
+    export interface PermissionOptionsObject {
+        /** Specifies if the `env` permission should be requested or revoked.
+         * If set to `"inherit"`, the current `env` permission will be inherited.
+         * If set to `true`, the global `env` permission will be requested.
+         * If set to `false`, the global `env` permission will be revoked.
+         *
+         * @default {false}
+         */
+        env?: "inherit" | boolean | string[];
```

Comment:
I think maybe adding it would add some confusion around what `env: undefined` means. Generally people would just provide the properties they want to change here.